packages/@stimulus/core/src/application.ts

```diff
@@ -41,6 +44,14 @@ export class Application implements ErrorHandler {
     this.load({ identifier, controllerConstructor })
   }
 
+  register_autoloader(autoloader: Autoloader) {
```

Comment:
I didn't feel this small addition of capability could justify changing the parameters to the Application constructor or the `start` wrapper. Even a class declaration seemed overkill. So I went with a basic registration/callback instead.